packages/cli/src/constants/router-contracts.ts

```diff
@@ -0,0 +1,37 @@
+import { Extension, ExtensionDeployArgs } from "../core/interfaces/Extension";
+
+export interface PreDeployedRouter {
+  name: string;
+  metadataUri: string;
+  bytecodeUri: string;
+}
+
+export interface RouterParams {
+  extensions: Extension[];
+  extensionDeployArgs: ExtensionDeployArgs[];
+}
+
+export interface TWRouterParams {
+  extensionRegistry: string;
+  extensionNames: string[];
+}
+
+export const EXTENSION_REGISTRY_ADDRESS: string = "0x0000000000000000000000000000000000000000";
+
+export const ROUTER_CONTRACTS: Record<string, PreDeployedRouter> = {
+  "upgradeable": {
+    name: "RouterUpgradeableFactory",
```

Comment:
when these show up in the CLI - we should have a little explanation of what they are next to each. Just like we do in `npx thirdweb detect`